artnet/plugin.yaml

```diff
@@ -35,36 +60,12 @@ parameters:
             de: 'Gibt das Art-Net Sub-Net an'
             en: 'Specifies the Art-Net Sub-Net to use'
         valid_min: 0
-        valid_max: 15
-    artnet_universe:
-        type: int
-        default: 0
-        description:
-            de: 'Gibt das Art-Net Universe an'
-            en: 'Specifies the Art-Net Universe to use'
-        valid_min: 0
-        valid_max: 15
-    ip:
-        type: ip
-        description:
-            de: 'Gibt die Adresse einer Art-Net Node an'
-            en: 'Specifies the address of a Art-Net node'
-    port:
+        valid_max: 15 
+
+item_attributes:
+    artnet_address:
         type: int
-        valid_min: 0
+        mandatory: True
         description:
-            de: 'Gibt den Port einer Art-Net Node an'
-            en: 'Specifies the port of a Art-Net node'
-
-item_attributes: NONE
-    # Definition of item attributes defined by this plugin
-
-item_structs: NONE
-  # Definition of item-structure templates for this plugin
-
-logic_parameters: NONE
-    # Definition of logic parameters defined by this plugin
-
-plugin_functions: NONE
-    # Definition of function interface of the plugin
-
```

Comment:
I think the attributes listed above (`item_structs`, `logic_parameters`, `plugin_functions`) should not be removed.

Reply:
OK, I based this on the information from the AVM plugin. no issue, this can be fixed.

Reply:
plugin_function is a definition currently used for autocomplete in the logics editor.
item_attributes are for future version of the core.

NONE should be correct here. do i miss sth?